### PR TITLE
[MNO-281] Add intercom setting

### PR DESCRIPTION
**Pull request by hedudelgado**
@ouranos 

Have a look at the email I sent.

### CODE REVIEW

**Comment by hedudelgado** [comment]
@ouranos Have a look at this when you have time please :) thanks!

**Comment by hedudelgado** [comment]
https://github.com/maestrano/mno-enterprise-angular/pull/52

**Comment by hedudelgado** [comment]
It has been updated @ouranos .

File: core/lib/generators/mno_enterprise/install/templates/config/application.yml [hunk 1]
```diff
@@ -13,3 +13,8 @@ SECRET_KEY_BASE: <%= SecureRandom.hex(64) %>
 
 # Google Tag Manager
 google_tag_container: GTM-TH3MLB
+
+# Intercom(Uncomment and Add a valid intercom_id to enable)
+# intercom_id: intercom_id
```
Comment (ouranos) [inline core/lib/generators/mno_enterprise/install/templates/config/application.yml]: the variable names should be upper case to match with your initializer

**Comment by ouranos** [comment]
@hedudelgado all good, just change the case in the sample file.

**Comment by hedudelgado** [comment]
@ouranos updated

**Pull request merged**